app/javascript/users-answers.vue

```diff
@@ -0,0 +1,111 @@
+<template lang="pug">
+div
+  nav.pagination(v-if='totalPages > 1')
+    pager(v-bind='pagerProps')
+  div(v-if='answers === null')
+    loadingListPlaceholder
+    .thread-list.a-card
+  .o-empty-message(v-else-if='answers.length === 0')
+    .o-empty-message__icon
+      i.far.fa-sad-tear
+    p.o-empty-message__text
+      | 回答はまだありません。
+  .thread-list.a-card(v-else)
+    .thread-list__items
+      usersAnswer(
+        v-for='answer in answers',
+        :key='answer.id',
+        :answer='answer'
+      )
+  nav.pagination(v-if='totalPages > 1')
+    pager(v-bind='pagerProps')
+</template>
+
+<script>
+import LoadingListPlaceholder from './loading-list-placeholder.vue'
+import Pager from './pager.vue'
+import UsersAnswer from './users-answer.vue'
+
+export default {
+  components: {
+    loadingListPlaceholder: LoadingListPlaceholder,
+    pager: Pager,
+    usersAnswer: UsersAnswer
+  },
+  props: {
+    usersPath: { type: String, default: '', required: false }
```

Comment:
```suggestion
    usersPath: { type: String, default: null, required: false }
```

ここのdefault値は`null`とした方が良いようです。
理由は、https://github.com/fjordllc/bootcamp/pull/4293#discussion_r823500400 を参照ください。